2_configure/README.md

```diff
@@ -53,3 +53,7 @@ by playbooks. So, instead of invoking `ansible-playbook` directly to configure t
 ```
 ../ansible_common/ap.sh configure.yml
 ```
+
+## Helpful resources
+* [OCI docs - Ansible](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/ansible.htm)
```

Comment:
Can you replace "docs" with "documentation"? Because I'm old, fussy and petty.

Reply:
Will do! I'll go ahead and add some resources to the other READMEs as well.